demisto_sdk/commands/common/hook_validations/content_entity_validator.py

```diff
@@ -277,13 +277,33 @@ def are_tests_registered_in_conf_json_file_or_yml_file(self, test_playbooks: lis
 
         # Integration case
         elif file_type == 'integration':
-            is_configured_test = any(
-                test_config for test_config in conf_json_tests if is_test_config_match(test_config,
-                                                                                       integration_id=content_item_id))
+            is_configured_test = False
+            unconfigured_test_playbook_ids = []
+
+            if test_playbooks:
+                configured_tests = []
+                for test_playbook in test_playbooks:
+                    test_config_matches = []
+                    for test_config in conf_json_tests:
+                        test_config_matches.append(is_test_config_match(test_config,
+                                                                        test_playbook_id=test_playbook,
+                                                                        integration_id=content_item_id))
```

Comment:
Here we have a philosophical question,
if we have `integration1` with 
```
tests: 
 - some-test
 ```
 in the yml, but not in the conf.json.
 But, some other integration, `integration2` uses the same test `some-test` and `some-test` is configured in conf.json with it
 ```
 {
      "integrations": "integration2",
      "playbookID": "some-test"
 }
 ```
 Should the validation fail? After all, the test is present in the conf.json and will run on nightly. 
 I believe (and implemented) that it should fail, since collect_tests on integration change, only collects tests from config.json rather then the yml
 
 From collect_tests.py:
 ```
                 else:
                    # integration to test mapping available, and support level == xsoar (so - we run the tests)
                    tests = tuple(self.conf.integrations_to_tests[yml.id_])
                reason = CollectionReason.INTEGRATION_CHANGED
 ```

Reply:
yes